src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -22,44 +33,49 @@
 "_comment": "STATIONS",
 
 	"block.mmorpg.teleporter": "Картографическое устройство",
+	"block.mmorpg.station.cooking": "Станция готовки",
+	"block.mmorpg.station.salving": "Станция утилизации",
+	"block.mmorpg.station.gear_crafting": "Станция снаряжения",
+	"block.mmorpg.station.alchemy": "Алихимическая станция",
+	"block.mmorpg.station.enchanting": "Модификационная станция",
 
 "_comment": "ITEM TOOLTIPS",
 
-	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции для снаряжения",
-	"mmorpg.tooltip.lootbox": "Что может быть внутри?",
+	"mmorpg.tooltip.item_modifiable_in_station": "Можно использовать внутри модификационной станции",
+	"mmorpg.tooltip.lootbox": "Что внутри?",
 	"mmorpg.tooltip.player_levelup": "Если ваша шкала опыта заполнена, используйте этот предмет для повышения уровня!",
 	"mmorpg.tooltip.capacitor": "Поместите в станцию ремонта, чтобы уменьшить потребление топлива.",
 	"mmorpg.tooltip.capacitor2": "Множитель топлива",
-	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в гнездо",
+	"mmorpg.tooltip.socketable_item": "Этот самоцвет можно вставить в свободное гнездо",
 
 "_comment": "RARITY ITEM NAMES",
 
 "_comment": "MISC ITEMS",
 
 	"item.mmorpg.jewel.dex": "Изумрудный самоцвет",
 	"item.mmorpg.jewel.str": "Метеоритный самоцвет",
-	"item.mmorpg.jewel.int": "Звездный самоцвет",
-	"item.mmorpg.jewel.unique_crafted": "Уникальный самоцвет рунического мастерства",
+	"item.mmorpg.jewel.int": "Звездная пыль",
+	"item.mmorpg.jewel.unique_crafted": "Уникальный самоцвет Рунной мастерской",
 	"item.mmorpg.stone.0": "Обычный камень",
 	"item.mmorpg.stone.1": "Необычный камень",
 	"item.mmorpg.stone.2": "Редкий камень",
 	"item.mmorpg.stone.3": "Эпический камень",
 	"item.mmorpg.stone.4": "Легендарный камень",
 	"item.mmorpg.stone.5": "Мифический камень",
-	"item.mmorpg.map": "Карта приключений",
+	"item.mmorpg.map": "Карта подземелья",
 	"item.mmorpg.stat_soul": "Душа снаряжения",
-	"item.mmorpg.set_map": "Утилита настройки карт",
-	"item.mmorpg.loot_bag": "Сумка снаряжения",
-	"item.mmorpg.map_bag": "Сумка с картами",
-	"item.mmorpg.currency_bag": "Сумка с валютой",
+	"item.mmorpg.set_map": "Карта установки",
+	"item.mmorpg.loot_bag": "Сумка добычи снаряжения",
+	"item.mmorpg.map_bag": "Сумка карт",
+	"item.mmorpg.currency_bag": "Сумка валюты",
 	"item.mmorpg.newbie_gear_bag": "Сумка новичка",
-	"item.mmorpg.tutorial_book": "Учение об Эпохе Изгнания",
-	"item.mmorpg.map_back_portal": "Жемчуг отхода с карты",
+	"item.mmorpg.tutorial_book": "Учебник эпохи изгнания",
+	"item.mmorpg.map_back_portal": "Жемчуг выходи из подземелья",
```

Ты опять невнимательно сам себя проверил, ппц
```suggestion
	"item.mmorpg.map_back_portal": "Жемчуг выхода из подземелья",
```

---

Исправлено